yarn.lock

```diff
@@ -4075,9 +4075,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@openmrs/esm-api@npm:5.5.1-pre.1659":
-  version: 5.5.1-pre.1659
-  resolution: "@openmrs/esm-api@npm:5.5.1-pre.1659"
+"@openmrs/esm-api@npm:5.5.1-pre.1685":
```

Comment:
Updated framework because the previous version had an issue with overriding importmaps using the dev tools